ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -874,17 +874,29 @@ function getPoint(
         return boostPoint as BoostPointComposition;
     }
 
-    const xData = (
+    const isScatter = series.type === 'scatter',
+        xData = (
+            (isScatter && series.getColumn('x', true).length ?
+                series.getColumn('x', true) :
+                void 0) ||
             (series.getColumn('x').length ? series.getColumn('x') : void 0) ||
             seriesOptions.xData ||
             series.getColumn('x', true) ||
             false
         ),
+        yData = (
+            series.getColumn('y', true) ||
+            series.getColumn('y') ||
```

Comment:
This condition is probably redundant, because if procesed data is not available, `series.getColumn` will return the unprocessed. Don't take my word for it though, it should be tested.
```suggestion
            series.getColumn('y', true)
```

Reply:
Based on inspection of the code and testing, your assumption seems correct. 
I have changed this – thank you.